// @unified-latex/unified-latex-ctan/index.js
import { macros, environments } from "./package/cleveref/index.js";
import { m as macros$1, e as environments$1 } from "./provides-bdAvbanT.js";
import { macros as macros$2, environments as environments$2 } from "./package/geometry/index.js";
import { macros as macros$3, environments as environments$3 } from "./package/hyperref/index.js";
import { m as macros$4, e as environments$4 } from "./provides-oMoMsm7a.js";
import { m as macros$5, e as environments$5 } from "./provides-CyrbEVlM.js";
import { macros as macros$6, environments as environments$6 } from "./package/makeidx/index.js";
import { macros as macros$7, environments as environments$7 } from "./package/mathtools/index.js";
import { m as macros$8, e as environments$8 } from "./provides-RZ6B8tPN.js";
import { macros as macros$9, environments as environments$9 } from "./package/nicematrix/index.js";
import { m as macros$a, e as environments$a } from "./provides-FG1zhZZq.js";
import "@unified-latex/unified-latex-util-match";
import "@unified-latex/unified-latex-util-pegjs";
import "@unified-latex/unified-latex-util-print-raw";
import "@unified-latex/unified-latex-util-split";
import "./index-_2u1fzdR.js";
import "@unified-latex/unified-latex-util-comments";
import "@unified-latex/unified-latex-util-visit";
import "@unified-latex/unified-latex-util-render-info";
import "@unified-latex/unified-latex-util-arguments";
import { m as macros$b, e as environments$b } from "./parser-Nnm59xBV.js";
import "@unified-latex/unified-latex-util-trim";
import { m as macros$c, e as environments$c } from "./xcolor-loVqJDRq.js";
import "@unified-latex/unified-latex-builder";
import { macros as macros$d, environments as environments$d } from "./package/xparse/index.js";
import { macros as macros$e, environments as environments$e } from "./package/beamer/index.js";
import { macros as macros$f, environments as environments$f } from "./package/multicol/index.js";
const macroInfo = {
  cleveref: macros,
  exam: macros$1,
  geometry: macros$2,
  hyperref: macros$3,
  latex2e: macros$4,
  listings: macros$5,
  makeidx: macros$6,
  mathtools: macros$7,
  minted: macros$8,
  nicematrix: macros$9,
  systeme: macros$a,
  tikz: macros$b,
  xcolor: macros$c,
  xparse: macros$d,
  beamer: macros$e,
  multicol: macros$f
};
const environmentInfo = {
  cleveref: environments,
  exam: environments$1,
  geometry: environments$2,
  hyperref: environments$3,
  latex2e: environments$4,
  listings: environments$5,
  makeidx: environments$6,
  mathtools: environments$7,
  minted: environments$8,
  nicematrix: environments$9,
  systeme: environments$a,
  tikz: environments$b,
  xcolor: environments$c,
  xparse: environments$d,
  beamer: environments$e,
  multicol: environments$f
};
export {
  environmentInfo,
  macroInfo
};
